lab_1/main.py

```diff
@@ -173,7 +169,7 @@ def write_to_file(path_to_file: str, content: list):
     if isinstance(path_to_file, str) and isinstance(content, list):
         with open(path_to_file, 'w', encoding='utf-8') as file:
             for i in content:
-                file.write(" ".join(i) + "\n")
+                file.write("\n".join(i))
```

Comment:
please, put "i" in variable 'result', for example
and then OUT OF "for" write to file this "result"